README.rst

```diff
@@ -113,15 +114,22 @@ For example, to access the raw text from the AG_NEWS dataset:
       >>> train_iter = AG_NEWS(split='train')
       >>> dataloader = DataLoader(train_iter, batch_size=8, shuffle=False)
 
-A tutorial for the end-to-end text classification workflow can be found in `PyTorch tutorial <https://pytorch.org/tutorials/beginner/text_sentiment_ngrams_tutorial.html>`_
+Tutorials
+=========
+
+To get started with torchtext, users may refer to following tutorials available on PyTorch website.
```

Comment:
nit: "users may refer to *the* following tutorials" sounds more correct imo (keep in mind I'm also ESL).